### ISSUE TITLE
Document how to restart after changing environment variables

### ISSUE THREAD

**Issue by gioragutt** [zulip/docker-zulip#128]
Hi, consider I'm accessing the zulip container with something like
```bash
sudo docker-compose exec zulip /bin/bash
```
And I want to f.e edit `/etc/zulip/settings.py`, currently the two methods I had were:
1. Add a working debian souce and apt-get it myself - takes a bit more, easily overriden
2. `docker cp` out -> edit the file -> `docker cp` in.

Not really comfortable, the second one is more reliable, but the best thing would be to easily edit the files with `vim` when you have an open bash in the container.

**Comment by timabbott** [comment]
@galexrt do you have thoughts on what folks normally do for editors in an image like this?  They're pretty small in disk size compared to the rest of the image, so possibly we should just include them, but I'm curious whether folks try to do it under an option or something.

**Comment by gioragutt** [comment]
@galexrt definitely not document. I would much rather have it linked outside and edit it how I like than having to vim inside the container. But compared to not being able to edit without `docker cp`ing, then vim is a better solution.

Can you elaborate on the env var here a it?

**Comment by galexrt** [comment]
@gioragutt If you set `LINK_SETTINGS_TO_DATA: "True"`, your data dir on the host will have a `settings` directory in which you can edit the `settings.py`.

**Comment by gioragutt** [comment]
Amazing. BTW, using docker-compose, what's your suggested method of restarting the server after changing env vars, without losing data? I do it every now and then, always hope mother gets ruined, and yesterday I fucked my server up and lost some data trying to make it work again.

**Comment by galexrt** [comment]
Just use `docker-compose stop && docker-compose rm` and to start again `docker-compose up -d`.

The container is currently for being started and "on stop" being deleted and recreated.

**Comment by gioragutt** [comment]
Yes. So the step I'm not doing is the ```rm```.

**Comment by gioragutt** [comment]
This should be added to the docs maybe. "How to restart the server after environment variables changes". Just to be sure, the ```rm``` does not delete the volume data, right?

**Comment by timabbott** [comment]
Agreed, I'm reopening this as a documentation issue for covering that.

**Comment by galexrt** [comment]
@gioragutt `docker-compose rm` only removes the containers, as long as you have specified volumes for the data in the `docker-compose.yml` you are good.

**Comment by alexmv** [comment]
https://zulip.readthedocs.io/projects/docker/en/latest/how-to/compose-settings.html now documents `docker compose up`.
